service/debugger/debugger_linux.go

```diff
@@ -0,0 +1,29 @@
+package debugger
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"syscall"
+)
+
+func attachErrorMessage(pid int, err error) error {
+	fallbackerr := fmt.Errorf("could not attach to pid %d: %s", pid, err)
+	if serr, ok := err.(syscall.Errno); ok {
+		switch serr {
+		case syscall.EPERM:
+			bs, err := ioutil.ReadFile("/proc/sys/kernel/yama/ptrace_scope")
```

Comment:
Please add link to (https://www.kernel.org/doc/Documentation/security/Yama.txt) in comment above this line.